tgui/packages/tgui/interfaces/NtosPortraitPrinter.js

```diff
@@ -64,6 +104,7 @@ export const NtosPortraitPrinter = (props, context) => {
                       <Button
```

Comment:
Also, do you think you could fix how this button is really, but really off-centered?

Reply:
I'm not accustomed enough with TGUI to have a ready solution to it alas.